src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AddNewUserTest.java

```diff
@@ -68,7 +80,79 @@ public void should_throw_409_when_add_duplicate_new_user_to_organisation() {
         // now invite same user/email used in above pending org should give CONFLICT
         NewUserCreationRequest newUserCreationRequest = professionalApiClient.createNewUserRequest();
         newUserCreationRequest.setEmail(pendingOrganisationCreationRequest.getSuperUser().getEmail());
-        Map<String, Object> newUserResponse = professionalApiClient.addNewUserToAnOrganisation(organisationIdentifier, hmctsAdmin,newUserCreationRequest, HttpStatus.CONFLICT);
-        assertThat((String)newUserResponse.get("errorDescription")).contains("409 User already exists");
+        Map<String, Object> newUserResponse = professionalApiClient.addNewUserToAnOrganisation(organisationIdentifier, hmctsAdmin, newUserCreationRequest, HttpStatus.CONFLICT);
+        assertThat((String) newUserResponse.get("errorDescription")).contains("409 User already exists");
+    }
+
+    @Test
+    public void add_new_user_to_organisation_by_super_user() {
+        List<String> userRoles = new ArrayList<>();
+        userRoles.add("caseworker");
+        NewUserCreationRequest newUserCreationRequest = aNewUserCreationRequest()
+                .firstName("someName")
+                .lastName("someLastName")
+                .email(RandomStringUtils.randomAlphabetic(10) + "@hotmail.com".toLowerCase())
+                .roles(userRoles)
+                .jurisdictions(createJurisdictions())
+                .build();
+
+        Map<String, Object> newUserResponse = professionalApiClient.addNewUserToAnOrganisationExternal(newUserCreationRequest, generateSuperUserBearerToken(), HttpStatus.CREATED);
+        assertThat(newUserResponse).isNotNull();
+    }
+
+    @Ignore //currently being ignored due to response status code coming 500 from SIDAM - but comes 403 when testing in Swagger

```

Comment:
We can't keep any @ignored test cases , could you please dig down why the test case is failing . There is no actual SIDAM here.

Reply:
I don't understand what you mean by "no actual SIDAM" as this is a functional test?

Added @Ignore after discussions with @KOTLAPRA23